.github/workflows/pkgdown.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: pkgdown
+          extra-packages: any::pkgdown, local::.
```

Comment:
Fixing this whilst I'm as failing to install on main branch. This changed with change from v1 action to v2